es_managers/es_idx_manager/test/test_app.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import os
+import sys
+import unittest
+from copy import deepcopy
+import yaml
+from app import ESException, ESCleanup
+import datetime
+
+
+class TestApp(unittest.TestCase):
+
+    with open(os.environ['ES_IDX_MANAGER_SETTINGS'], 'r') as config_file:
+        config = yaml.load(config_file)
+
+    cluster_config = config[0]
+    es = ESCleanup(None, None, cluster_config)
+
+    def test_fetch_indices(self):
+        indices = self.es.get_indices()
+        indices_names = [index['index'] for index in indices]
+        # .kibana is always an index that is never deleted
+        self.assertEqual('.kibana' in indices_names, True)
+        # Should only ever have 7 days of logs + kibana
+        self.assertEqual(len(indices) <= 8, True)
```

Comment:
I think this test couples the state of production to this test too tightly. I think it might be cleaner to isolate this by configuring a different index prefix, creating > 7 indices, run these tests, and then actually run the full pipeline. This way you could confirm that you have _exactly_ the number of indices that you expect.

Reply:
That sounds like a much cleaner test! I'll update the PR